packages/test/test-service-load/src/runner.ts

```diff
@@ -96,6 +96,7 @@ async function main() {
 		driverType: driver,
 		driverEndpointName: endpoint,
 		profile: profileName,
+		ring: endpoint,
```

Comment:
This doesn't feel right. When `orchestratorProcess` from nodeStressTest.ts spawns `dist/runner.js`, it probably should pass `--ring` as a parameter to it as well, so we have the value here.